network/src/protocols/identity.rs

```diff
@@ -59,10 +59,11 @@ mod tests {
     #[test]
     fn simple_handshake() {
         let network_id = NetworkId::Validator;
-        let (mut outbound, mut inbound) = build_test_connection();
+        let chain_id = ChainId::default();
+        let (outbound, inbound) = build_test_connection();
 
         // Create client and server handshake messages.
-        let mut server_handshake = HandshakeMsg::new(network_id.clone());
+        let mut server_handshake = HandshakeMsg::new(network_id.clone(), chain_id.clone());
```

Comment:
my mental model is chain >> network
if you agree, let's make that ordering consistent

Reply:
Yeah, I think in general we should reorder a lot of these things to some sort of canonical order, and chain_id is definitely > network